zone/entity.h

```diff
@@ -240,11 +240,24 @@ class EntityList
 	bool IsMobInZone(Mob *who);
 	void ClearClientPetitionQueue();
 	bool CanAddHateForMob(Mob *p);
-	void	SendGuildMOTD(uint32 guild_id);
-	void	SendGuildSpawnAppearance(uint32 guild_id);
-	void	SendGuildMembers(uint32 guild_id);
+	void SendGuildMOTD(uint32 guild_id);
+	void SendGuildChannel(uint32 guild_id);
+	void SendGuildURL(uint32 guild_id);
+	void SendGuildSpawnAppearance(uint32 guild_id);
+	void SendGuildMembers(uint32 guild_id);
+	void SendGuildMembersList(uint32 guild_id);
+	void SendGuildMemberAdd(uint32 guild_id, uint32 level, uint32 _class, uint32 rank, uint32 spirit, uint32 zone_id, std::string player_name);
+	void SendGuildMemberRename(uint32 guild_id, std::string player_name, std::string new_player_name);
```

Comment:
![image](https://github.com/EQEmu/Server/assets/3319450/c389557a-835f-46cc-9ea2-4cb334bd13ca)

All of these in gray are un-used - what's the story here?

Reply:
These functions were added as I worked through the individual opcodes within RoF2.  The foundation for the guild code is to simply resend the entire guild list for every action.  This allows for individual packets to be sent, reducing network load considerably.  I therefore added these for future work to redo the underlying foundation to not send the entire guild list.  These are currently not yet implemented, though perhaps I should?  Or do as a phase 2?  Please let me know your thoughts.

Reply:
As long as you have intention to follow up on using them - I'm fine with them going in

Reply:
Can't get rid of me that easily.  :-)